client/src/App.js

```diff
@@ -1,22 +1,23 @@
 import React, { Component } from 'react';
 import './css/ResultsTable.css';
 import Navbar from './views/Navbar';
-import ResultData from './components/ResultData';
 import FilterMenu from './components/FilterMenu';
 import Routes from './config/Routes';
+import { withRouter } from 'react-router-dom';
 
 class App extends Component {
 
     constructor(props, context) {
         super(props, context);
 
         this.state = {
-            data: [
-                new ResultData("Nothing do-er", "nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop ", "nop nop nop nop nop nop nop nop nop ", "Vancouver, BC, Canada", "http://hasthelargehadroncolliderdestroyedtheworldyet.com/", "https://thedailywtf.com/articles/you-can-only-get-what-you-have"),
-                new ResultData("CPU maker", "Make CPUs and stuff. You also do things that make this string longer, so I can test how well this works for long strings. Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong ", "Make CPUs and stuff.",  "Vancouver, WA, USA", "https://ded.increpare.com/~locus/yourname.html", "https://ded.increpare.com/~locus/yourname.html"),
-                new ResultData("Software Developer", "Develop Software, but more text", "Develop Software", "Vancouver, BC, Canada", "https://github.com/mattdiamond/fuckitjs", "https://github.com/ajalt/fuckitpy")
-            ],
-            isFilterMenuVisible: false
+            data: [],
+            // data: [
```

Comment:
I just deleted the commented code entirely and didn't put it into another file. I couldn't find a place I thought made sense for it, and I figured that the data isn't that hard to generate since it was all made up and there wasn't that much of it. Plus, stuff may change before we write tests, and we can also easily go back and copy the data in previous commits if needed.